common/app/views/support/ContentFooterContainersLayout.scala

```diff
@@ -23,7 +23,8 @@ object ContentFooterContainersLayout {
     val htmlBlocks = if (isAdvertisementFeature) {
 
       // majority of footer components we don't want to appear on advertisement feature articles
-      Seq(storyPackagePlaceholder)
+      if (showPaidSeriesContainer.isSwitchedOn) Seq(storyPackagePlaceholder, onwardPlaceholder)
+      else Seq(storyPackagePlaceholder)
```

Comment:
Hmm, yes I think you're right. If we used the `++` operator instead then it would auto serialise the `None` to `Seq.empty`, but not sure what would happen to the HTML object.

Ignore me!